app_dart/lib/src/request_handlers/vacuum_github_commits.dart

```diff
@@ -35,13 +35,11 @@ class VacuumGithubCommits extends ApiRequestHandler<Body> {
 
   @override
   Future<Body> get() async {
```

Comment:
go ahead and remove the other floating datastore references in the file?  Its not used by _vacuumRepository and _vacuumBranch

Reply:
Done